indra/llcommon/llleaplistener.cpp

```diff
@@ -80,11 +80,13 @@ LLLeapListener::LLLeapListener(const std::string_view& caller, const Callback& c
     add("listen",
         "Listen to an existing LLEventPump named [\"source\"], with listener name\n"
         "[\"listener\"].\n"
+        "If [\"tweak\"] is specified as true, tweak listener name for uniqueness.\n"
         "By default, send events on [\"source\"] to the plugin, decorated\n"
         "with [\"pump\"]=[\"source\"].\n"
         "If [\"dest\"] specified, send undecorated events on [\"source\"] to the\n"
         "LLEventPump named [\"dest\"].\n"
-        "Returns [\"status\"] boolean indicating whether the connection was made.",
+        "Returns [\"status\"] boolean indicating whether the connection was made,\n"
+        "plus [\"listener\"] reporting (possibly tweaked) listener name.",
```

Comment:
That's a lot of quoting and finger gymnastics. Raw string literals (https://en.cppreference.com/w/cpp/language/string_literal) could make this string a lot more pleasant to type and read

Reply:
https://github.com/secondlife/viewer/pull/1110/commits/3f876a6a1d138c31266afb6c39df7090e304efe3